aexpect/ops.py

```diff
@@ -424,23 +424,29 @@ def grep(session, expr, path, check=False, flags="-a"):
 ###############################################################################
 
 
-def hash_file(session, filename, method='md5'):
+def hash_file(session, filename, method="md5", first_chunk=""):
```

Comment:
Naming is always hard and I am not against "first_chunk" but what would you say about `size` argument instead?

Reply:
Sounds great and even more compatible with the `avocado.utils.crypto` API (`def hash_file(filename, size=None, algorithm="md5")` but do you mind if I also add size as a second argument to achieve full compliance? I know it is not advised to change the order of previously defined arguments but considering this is in early stages of usages perhaps I could still take the jump?

Reply:
Sure, I do consider this module in alpha stage, still hammering out the APIs. Thanks for spotting this out and feel free to adopt the API from the `avocado.utils.crypto` module.